widget/ui/src/components/Select/Select.types.ts

```diff
@@ -0,0 +1,10 @@
+type Item = {
+  value: string;
+  label: string;
+};
+export interface PropTypes {
+  options: Item[];
+  handleItemClick?: (item: Item) => void;
+  value: Item;
+  container?: HTMLElement;
+}
```

Comment:
Prefer using the type keyword over interface to avoid declaration merging.

Reply:
fixed